classes/requests/helpers/class-nets-easy-checkout-helper.php

```diff
@@ -48,6 +48,7 @@ public static function get_checkout( $checkout_flow = 'embedded', $order_id = nu
 			$order                                   = wc_get_order( $order_id );
 			$checkout['returnUrl']                   = add_query_arg( 'easy_confirm', 'yes', $order->get_checkout_order_received_url() );
 			$checkout['integrationType']             = 'HostedPaymentPage';
+			$checkout['cancelUrl']                   = wc_get_checkout_url();
```

Comment:
Only works as long as the customer is using the checkout to create the order. If for example the customer is paying for the order using the order pay link for a admin created order, this will redirect them to an empty cart page.

Test this with Pay for order, and add support to be able to handle the case as well